pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -869,6 +869,48 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 				}: 1,
 			},
 		},
+		{
+			name: "Data frame header sent separately from frame payload",
```

Comment:
Let's add another use-case of data payload split
message 1: headers frame header, headers payload, data frame header, 10 bytes of data frame payload
message 2: the rest of the format data frame payload, another headers, another empty data